addon/components/rdf-input-fields/currency-input.js

```diff
@@ -0,0 +1,40 @@
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import SimpleInputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/simple-value-input-field';
+import { stringAmountToNumber } from '../../utils/string-amount-to-number';
+
+export default class RdfInputFieldsCurrencyInputComponent extends SimpleInputFieldComponent {
+  inputId = 'input-' + guidFor(this);
+
+  currencyIsocode = 'EUR';
+  decimalSeparator = ',';
+  thousandSeparator = '.';
+
+  @action
+  updateValue(e) {
+    if (e && typeof e.preventDefault === 'function') e.preventDefault();
+
+    if (e.target && e.target.value) {
+      const amount = stringAmountToNumber(e.target.value, {
+        decimalSeparator: this.decimalSeparator,
+        thousandSeparator: this.thousandSeparator,
+      });
+      this.value = this.numberStringToFormattedString(amount);
```

Comment:
I cannot get it to work with your setup. Did you change the decimals in the input field or also the fractions? I noticed that it does work if you only change the decimals and not the fractions. 

When reading the docs it also seems to be not possible to have an `inputType="number"` with a `radixPoint = ","`
![image](https://github.com/lblod/ember-submission-form-fields/assets/36266973/aa864cad-d2d0-47e9-bdb0-8675da98d219)

Reply:
I only changed the inputType value, and made sure that `this.value` was set to the number I got from `e.target.inputmask.unmaskedvalue()`. I'll create a PR with the changes so we're looking at the same code.

Reply:
This is the code that _seems_ to be working on first glance: https://github.com/lblod/ember-submission-form-fields/pull/176

Reply:
> When reading the docs it also seems to be not possible to have an inputType="number" with a radixPoint = ","

I _think_ this is talking about the value that you `.set` on the form field, so not the values that users enter by typing. That's how I understood it at least. So if you set `inputType` to number, you need to provide a number instead of the string when using `.set` (which Ember does when you pass it as the `value={{this.value}}` attribute.

Reply:
Strange, it seems we _are_ still passing a string because of the `formattedValue` getter, but it still worked on my machine 😄. Changing it to `this.value` directly also seems to work. 🤔 

Let me try in Chrome.

Reply:
> Strange, it seems we _are_ still passing a string because of the `formattedValue` getter, but it still worked on my machine 😄. Changing it to `this.value` directly also seems to work. 🤔
> 
> Let me try in Chrome.
> 
> Edit, also works in Chrome.

Did you just change the input value to this.value instead of the getter get formattedValue?